docs/config.md

```diff
@@ -114,47 +114,46 @@ ui.graph.style = "square"
 
 ### Display of commit and change ids
 
-```toml
-ui.unique-prefixes = "brackets"  # Does not rely on color
-```
-
-Whether to highlight a unique prefix for commit & change ids. Possible
-values are `styled`, `brackets` and `none` (default: `styled`).
+Can be customized by the `format_short_id()` template alias.
 
 ```toml
-ui.log-id-preferred-length = 6
+[template-aliases]
+# Highlight unique prefix and show at least 12 characters (default)
+'format_short_id(id)' = 'id.shortest(12)'
+# Just unique prefix
+'format_short_id(id)' = 'id.shortest()'
+# Show unique prefix and the rest surrounded by brackets
+'format_short_id(id)' = 'id.shortest(12).prefix() "[" id.shortest(12).rest() "]"'
```

Comment:
Not sure if it's worth it, but you could have

     id.shortest(12).prefix() if(id.shortest(12).rest(),"[" id.shortest(12).rest() "]"')

here. I suppose that is one thing that having scoped variables or some optimization would help with (though, it's certainly not worth doing for this one reason alone).

Reply:
Yeah, I decided to not include `if(...)` because this version looked too complicated.
We'll need a better syntax for that.